tests/resource/aws_fsx_lustre_file_system: RequiredWith Testing Errors

<!---
Please note the following potential times when an issue might be in Terraform core:

* [Configuration Language](https://www.terraform.io/docs/configuration/index.html) or resource ordering issues
* [State](https://www.terraform.io/docs/state/index.html) and [State Backend](https://www.terraform.io/docs/backends/index.html) issues
* [Provisioner](https://www.terraform.io/docs/provisioners/index.html) issues
* [Registry](https://registry.terraform.io/) issues
* Spans resources across multiple providers

If you are running into one of these scenarios, we recommend opening an issue in the [Terraform core repository](https://github.com/hashicorp/terraform/) instead.
--->

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### Terraform CLI and Terraform AWS Provider Version

Terraform 0.12.29 and Terraform AWS Provider `HEAD`

### Affected Resource(s)

* aws_fsx_lustre_file_system

### Terraform Configuration Files

`aws/resource_aws_fsx_lustre_file_system_test.go`

### Expected Behavior

All tests passing. 😄 

### Actual Behavior

```
=== CONT  TestAccAWSFsxLustreFileSystem_ExportPath
TestAccAWSFsxLustreFileSystem_ExportPath: resource_aws_fsx_lustre_file_system_test.go:143: Step 1/3 error: terraform failed: exit status 1
stderr:
Error: RequiredWith
on config907186540/terraform_plugin_test.tf line 28, in resource "aws_fsx_lustre_file_system" "test":
28:   import_path      = "s3://${aws_s3_bucket.test.bucket}"
"import_path": all of `auto_import_policy,import_path` must be specified
--- FAIL: TestAccAWSFsxLustreFileSystem_ExportPath (1.66s)

=== CONT  TestAccAWSFsxLustreFileSystem_ImportPath
TestAccAWSFsxLustreFileSystem_ImportPath: resource_aws_fsx_lustre_file_system_test.go:181: Step 1/3 error: terraform failed: exit status 1
stderr:
Error: RequiredWith
on config629149576/terraform_plugin_test.tf line 27, in resource "aws_fsx_lustre_file_system" "test":
27:   import_path      = "s3://${aws_s3_bucket.test.bucket}"
"import_path": all of `auto_import_policy,import_path` must be specified
--- FAIL: TestAccAWSFsxLustreFileSystem_ImportPath (1.62s)

=== CONT  TestAccAWSFsxLustreFileSystem_ImportedFileChunkSize
TestAccAWSFsxLustreFileSystem_ImportedFileChunkSize: resource_aws_fsx_lustre_file_system_test.go:217: Step 1/3 error: terraform failed: exit status 1
stderr:
Error: RequiredWith
on config185468749/terraform_plugin_test.tf line 27, in resource "aws_fsx_lustre_file_system" "test":
27:   import_path              = "s3://${aws_s3_bucket.test.bucket}"
"import_path": all of `auto_import_policy,import_path` must be specified
--- FAIL: TestAccAWSFsxLustreFileSystem_ImportedFileChunkSize (1.51s)
```

### Steps to Reproduce

1. `TF_ACC=1 go test -run=TestAccAWSFsxLustreFileSystem_ -v`

### References

* https://github.com/terraform-providers/terraform-provider-aws/pull/15231

--- a/aws/resource_aws_fsx_lustre_file_system.go
+++ b/aws/resource_aws_fsx_lustre_file_system.go
@@ -56,7 +56,6 @@ func resourceAwsFsxLustreFileSystem() *schema.Resource {
 					validation.StringLenBetween(3, 900),
 					validation.StringMatch(regexp.MustCompile(`^s3://`), "must begin with s3://"),
 				),
-				RequiredWith: []string{"auto_import_policy"},
 			},
 			"imported_file_chunk_size": {
 				Type:         schema.TypeInt,

